WordPress/Sniffs/WP/I18nSniff.php

```diff
@@ -103,6 +103,9 @@ public function process( PHP_CodeSniffer_File $phpcs_file, $stack_ptr ) {
 		if ( ! empty( self::$text_domain_override ) ) {
 			$this->text_domain = self::$text_domain_override;
 		}
+		if ( is_string( $this->text_domain ) ) {
+			$this->text_domain = explode( ',', $this->text_domain );
```

Comment:
You might want to add a `array_map( 'trim', $this->text_domain);` after this to allow for spaces in the array value like:
```xml
	    <properties>
	        <property name="text_domain" value="default, _s, tgmpa" type="array" />
	    </properties>
```